### PR TITLE
[Feat][UI] Add swipe gestures open/close the sidebar

### PR DESCRIPTION
**Pull request by LouisLibre**
This PR adds support to open/close the sidebar using the swipe right or swipe left gesture.

Demo:

https://user-images.githubusercontent.com/115233399/232254918-a6ab74da-7100-4f81-ae9e-2009336bcd6b.mp4

### CODE REVIEW

File: damus/Info.plist [hunk 1]
```diff
@@ -46,9 +46,5 @@
 		<key>NSAllowsArbitraryLoads</key>
 		<true/>
 	</dict>
-	<key>NSCameraUsageDescription</key>
```
Comment (tyiu) [inline damus/Info.plist:49]: Accidentally removed?

Comment (LouisSaberhagen) [reply damus/Info.plist:49]: Yes. Thanks for catching that up

**Comment by LouisSaberhagen** [comment]
I tested every other gesture manually to make sure theres no conflict.

- [x] Timeline Post<->Post&Replies CustomPicker
- [x] Timeline Events Image Carousel's
- [x] Direct Messages DMs<->Requests CustomPicker
- [x] Notifications All<->Zaps<->Mentions CustomPicker
- [x] SwipeToDismiss Modifier
- [x] Zap ButtonLong Presses
- [x] All regular tap gestures working

**Comment by jb55** [comment]
I want this for all navigation not just sidebar opening ...

**Comment by jb55** [comment]
I've closed these due to inactivity, please re-opened if you want me to review again.

**Pull request closed without merging**